core/directory.go

```diff
@@ -741,7 +740,7 @@ func (dir *Directory) Export(ctx context.Context, destPath string, merge bool) (
 	}
 
 	var defPB *pb.Definition
-	if dir.Dir != "" {
+	if dir.Dir != "" && dir.Dir != "/" {
```

Comment:
not technically related to filesync, just noticed a bunch of extra `copy / /` steps when running exports and fixed that here